### PR TITLE
Kapacitor Server

### PR DESCRIPTION
**Pull request by tonerdo**
This PR does the following:

Add the `kapacitor-server` Terraform module that deploys a single EC2 instance in an ASG to allow for auto-recovery. Also updated the README to indicate the switch to the OSS edition
Add the `kapacitor-security-group-rules` Terraform module to allow opening up inbound requests on the HTTP port the server (deployed by the module above) listens on for requests

### CODE REVIEW

**Comment by tonerdo** [comment]
Merging so I can go ahead to the examples

File: modules/kapacitor-server/main.tf [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "aws_autoscaling_group" "autoscaling_group" {
+  name = "${var.server_name}"
+
+  launch_configuration = "${aws_launch_configuration.launch_configuration.name}"
+  vpc_zone_identifier  = ["${var.subnet_ids}"]
+
+  min_size             = "1"
+  max_size             = "1"
```
Comment (brikis98) [inline modules/kapacitor-server/main.tf]: Don't need quotes around numbers. Also, why just a single server? Can this not be run as a cluster with a load balancer?

Comment (tonerdo) [reply modules/kapacitor-server/main.tf]: I had a conversation with the InfluxSata team. We decided that open source Kapacitor was the way to go. Clustering capabilities are exclusive to the Enterprise edition

Comment (brikis98) [reply modules/kapacitor-server/main.tf]: Ah, right. Might be worth adding a comment about that.

**Pull request merged**